### PR TITLE
Add output to Step Function on failure

### PR DESCRIPTION
**Pull request by stekern**
- Outputs the last 15 lines of stdout and stderr to the Step Function task on failure

### CODE REVIEW

File: src/run-single-fargate-task.py [hunk 1]
```diff
@@ -61,6 +61,15 @@ def create_task_definition(
     client = boto3.client("ecs")
     task_family = f"{state.replace(' ', '_') if state else 'one-off-task'}-{date_time_obj.strftime('%Y%m%d%H%M')}"
     shellscript = (
+        "cat <<EOF >> /tmp/workspace/error_header.log\n"
+        "---------------\n"
+        "THE FOLLOWING IS JUST AN EXCERPT - FULL LOG AVAILABLE AT:\n"
+        "\n"
+        f"https://eu-west-1.console.aws.amazon.com/cloudwatch/home?region=eu-west-1#logStream:group=/aws/ecs/{task_name};prefix={task_family}-main;streamFilter=typeLogStreamPrefix\n"
```
Comment (colincoleman) [inline src/run-single-fargate-task.py]: No guarantee that this is running in eu-west-1

File: src/run-single-fargate-task.py [hunk 2]
```diff
@@ -192,6 +202,7 @@ def prepare_cmd(content, token):
             + token
             + ' --task-output \'{"output": "$result"}\' --region eu-west-1; else aws stepfunctions send-task-failure --task-token '
```
Comment (colincoleman) [inline src/run-single-fargate-task.py]: Same here (eu-west-1 assumed) although I guess we didn't spot this on the last merge

File: src/run-single-fargate-task.py (hunk 1 above)
Comment (stekern) [reply src/run-single-fargate-task.py]: That's true, but eu-west-1 is hardcoded into the rest of the file as well, and I think it always has been.. I can fix that here, or we can do that in another PR

**Review by colincoleman — APPROVED** [review]
Looks good - nice improvement

**Pull request merged**